src/pybind/mgr/dashboard/tools.py

```diff
@@ -766,6 +766,16 @@ def str_to_bool(val):
     return bool(strtobool(val))
 
 
+def json_str_to_dict(value):
+    if value is not None and value != '':
+        return json.loads(value.decode('utf-8'))
+    return value
```

Comment:
Why not setting [our default encoding to UTF-8](https://stackoverflow.com/questions/2276200/changing-default-encoding-of-python)?

Reply:
Interesting topic (would this have any unexpected side effects?)
but IMHO this belongs to another refactor/PR.

Reply:
In this particular case, ```.decode('utf-8')``` seems it is inheritance for compatibility with older python versions in order to deal with bytes strings, but in this particular case it is no longer needed, so it is removed.